### ISSUE TITLE
SimpleRoundIconButton: onPressed() not working

### ISSUE THREAD

**Issue by enricobenedos** [samarthagarwal/FlutterScreens#11]
Good Morning,

I'm trying to use the beautiful SimpleRoundButton but I'm not able to keep it working when pressed.
This is my code:
```java
final loginButton = SimpleRoundIconButton(
      buttonText: Text('Log In', style: TextStyle(color: Colors.white)),
      icon: const Icon(Icons.fingerprint),
      backgroundColor: globals.fcsBlue,
      iconAlignment: Alignment.centerRight,
      onPressed: () async {
        setState(() {
          isPageBusy = true;
        });
        //All my logic here
      },
    );
```
The button is correctly visualized but when I press it doesn't work. I'm on stable Flutter channel with v1.0.0.
Can anyone help me?
Thank you

**Comment by samarthagarwal** [comment]
If you put in a simple `print("hello");` statement in the `onPressed` callback method, does it work?

**Comment by enricobenedos** [comment]
> If you put in a simple print("hello"); statement in the onPressed callback method, does it work?

In both the next configurations it doesn't work:

```java
final loginButton = SimpleRoundIconButton(
      buttonText: Text('Log In', style: TextStyle(color: Colors.white)),
      icon: const Icon(Icons.fingerprint),
      backgroundColor: globals.fcsBlue,
      iconAlignment: Alignment.centerRight,
      onPressed: () {
        print("Hello");
        //All my logic here
      },
    );
```
```java
final loginButton = SimpleRoundIconButton(
      buttonText: Text('Log In', style: TextStyle(color: Colors.white)),
      icon: const Icon(Icons.fingerprint),
      backgroundColor: globals.fcsBlue,
      iconAlignment: Alignment.centerRight,
      onPressed: () async {
        print("Hello");
        //All my logic here
      },
    );
```

If needed this is my testing Scaffold:
```java
return Scaffold(
      backgroundColor: Colors.white,
      body: isPageBusy
          ? /*new Row(
        mainAxisAlignment: MainAxisAlignment.center,
              children: <Widget>[
                new Column(
                  mainAxisAlignment: MainAxisAlignment.center,
                  crossAxisAlignment: CrossAxisAlignment.center,
                  children: <Widget>[
                    new CircularProgressIndicator(
                      valueColor:
                          new AlwaysStoppedAnimation<Color>(globals.fcsBlue),
                    ),
                    new Text("Loading")
                  ],
                )
              ],
            )*/
          new Row(
              mainAxisAlignment: MainAxisAlignment.center,
              children: <Widget>[
                new Column(
                  mainAxisAlignment: MainAxisAlignment.center,
                  crossAxisAlignment: CrossAxisAlignment.center,
                  children: <Widget>[
                    new ColorLoader2(
                      color1: globals.fcsLightBlue,
                      color2: globals.fcsBlue,
                      color3: globals.fcsLightBlue,
                    ),
                    new Text("Loading")
                  ],
                )
              ],
            )
          : Center(
              child: ListView(
                shrinkWrap: true,
                padding: EdgeInsets.only(left: 24.0, right: 24.0),
                children: <Widget>[
                  logo,
                  new Column(
                    children: <Widget>[
                      new Row(
                        children: <Widget>[
                          new Expanded(
                            child: new Padding(
                              padding: const EdgeInsets.only(left: 40.0),
                              child: new Text(
                                "Username",
                                style: TextStyle(
                                  fontWeight: FontWeight.bold,
                                  color: globals.fcsBlue,
                                  fontSize: 15.0,
                                ),
                              ),
                            ),
                          ),
                        ],
                      ),
                      new Container(
                        width: MediaQuery.of(context).size.width,
                        margin: const EdgeInsets.only(
                            left: 40.0, right: 40.0, top: 10.0),
                        alignment: Alignment.center,
                        decoration: BoxDecoration(
                          border: Border(
                            bottom: BorderSide(
                                color: globals.fcsBlue,
                                width: 0.5,
                                style: BorderStyle.solid),
                          ),
                        ),
                        padding: const EdgeInsets.only(left: 0.0, right: 10.0),
                        child: new Row(
                          crossAxisAlignment: CrossAxisAlignment.center,
                          mainAxisAlignment: MainAxisAlignment.start,
                          children: <Widget>[
                            new Expanded(
                              child: TextField(
                                controller: _userController,
                                obscureText: true,
                                textAlign: TextAlign.left,
                                decoration: InputDecoration(
                                  border: InputBorder.none,
                                  hintText: 'mariocontoso',
                                  hintStyle: TextStyle(color: Colors.grey),
                                ),
                              ),
                            ),
                          ],
                        ),
                      ),
                      new Row(
                        children: <Widget>[
                          new Expanded(
                            child: new Padding(
                              padding: const EdgeInsets.only(left: 40.0),
                              child: new Text(
                                "Password",
                                style: TextStyle(
                                  fontWeight: FontWeight.bold,
                                  color: globals.fcsBlue,
                                  fontSize: 15.0,
                                ),
                              ),
                            ),
                          ),
                        ],
                      ),
                      new Container(
                        width: MediaQuery.of(context).size.width,
                        margin: const EdgeInsets.only(
                            left: 40.0, right: 40.0, top: 5.0),
                        alignment: Alignment.center,
                        decoration: BoxDecoration(
                          border: Border(
                            bottom: BorderSide(
                                color: globals.fcsBlue,
                                width: 0.5,
                                style: BorderStyle.solid),
                          ),
                        ),
                        padding: const EdgeInsets.only(left: 0.0, right: 10.0),
                        child: new Row(
                          crossAxisAlignment: CrossAxisAlignment.center,
                          mainAxisAlignment: MainAxisAlignment.start,
                          children: <Widget>[
                            new Expanded(
                              child: TextField(
                                controller: _passwordController,
                                obscureText: true,
                                textAlign: TextAlign.left,
                                decoration: InputDecoration(
                                  border: InputBorder.none,
                                  hintText: '*********',
                                  hintStyle: TextStyle(color: Colors.grey),
                                ),
                              ),
                            ),
                          ],
                        ),
                      ),
                    ],
                  ),
                  loginButton,
                  forgotLabel
                ],
              ),
            ),
    );
```

**Comment by samarthagarwal** [comment]
I did not need all that code. Does that `Hello` get printed in the console when you tap on the button?

**Comment by enricobenedos** [comment]
No it isn't print

**Comment by samarthagarwal** [comment]
Fixed in [affdd36](https://github.com/samarthagarwal/FlutterScreens/commit/affdd36ae3bf301f0788cd36fbd98065184337a8).

**Comment by enricobenedos** [comment]
Thank you samarthagarwal
